Solutions/Marain.TenantManagement.Cli/Marain/TenantManagement/Cli/Commands/CreateClientTenantCommand.cs

```diff
@@ -27,16 +27,21 @@ public CreateClientTenantCommand(ITenantManagementService tenantManagementServic
                 Description = "The name of the new client.",
                 Arity = ArgumentArity.ExactlyOne,
             };
+            var parentId = new Option<string>("--parentId")
+            {
+                Description = "[Optional] The ID of the parent Client Tenant.",
+            };
 
             this.AddArgument(clientName);
+            this.AddOption(parentId);
 
-            this.Handler = CommandHandler.Create((string name) => this.HandleCommand(name));
+            this.Handler = CommandHandler.Create((string name, string parentId) => this.HandleCommand(name, parentId));
```

Comment:
Does `System.CommandLine` not like it if you use `string? parentId` here? I know it does some magic to map the params to the arguments/options, but I don't know how sensitive it is...

Reply:
Seems to be happy - changed